File: Day2/python/caesar.py
def encrypted_text( text, shift):
    result = ""
    
    
    shift = shift % 26
    
    for index in range(len(text)):
        letter = text[index]
        
        
        shifted_letter = chr(ord(letter) + shift)
        
        
        if letter.isupper() and shifted_letter > 'Z':
            shifted_letter = chr(ord(shifted_letter) - 26)
            
        
        elif letter.islower() and shifted_letter > 'z':
            shifted_letter = chr(ord(shifted_letter) - 26)
            
        result = result + shifted_letter
        
    return result

File: Day2/java/Caesar.java




public class Caesar{






public static String encryptedText(String text, int shift) {
    String result = ""; 

shift = shift % 26;

    for (int index = 0; index < text.length(); index++) {
        char letter = text.charAt(index);

       
        char shiftedLetter = (char) (letter + shift);

        
        if (Character.isUpperCase(letter) && shiftedLetter > 'Z') {
            shiftedLetter = (char) (shiftedLetter - 26); 
        }
        
        else if (Character.isLowerCase(letter) && shiftedLetter > 'z') {
            shiftedLetter = (char) (shiftedLetter - 26); 
        }

        
        result = result + shiftedLetter;
    }

    return result;
}







}
